harpgl/demo.js

```diff
@@ -1,15 +1,15 @@
            
            const map = new harp.MapView({
             canvas: document.getElementById("map"),
-            theme: "https://assets.vector.hereapi.com/styles/berlin/base/harp.gl/tilezen?apikey=xxx",
+            theme: "https://assets.vector.hereapi.com/styles/berlin/base/harp.gl/tilezen?apikey=EybNF4MkvcUj8WX1VJRKhT9dwDVz1wIHaKGF5tpqNss",
          });
    
          const controls = new harp.MapControls(map);
-         window.onresize = () => map.resize(window.innerWidth, window.innerHeight);
-         map.setCameraGeolocationAndZoom(
-            new harp.GeoCoordinates(37.773972, -122.431297), //Latitude, Longitude
-            11
+         window.onresize = () => map.resize(
+            document.getElementById("map").innerWidth,
+            document.getElementById("map").innerHeight
          );
+         map.lookAt(new harp.GeoCoordinates(37.773972, -122.431297), 14000, 0, 0);
```

Comment:
If the actual zoomlevel is important you could call map.zoomLevel = 11 afterwards (in the future it will be possible to pass it to the lookAt directly)

Reply:
I am not so sure if it is that important or not. I just want the harpgl demo to show roughly the same as mapbox and tangram